### DISCUSSION TITLE
Laravel 9 compatibility

### DISCUSSION THREAD

**Discussion by chuckgriffith2** [livewire/livewire#4864] [1. Help]
Well I'm not getting any feedback on the "Unable to find component" problem from anywhere in the world.

Now I'm wondering just how compatible is livewire with Laravel 9?
I can't even get the 'counter' example to work at the first step.
I see others getting further but I can't.
It dies in LifecycleManager->fromInitialRequest when it calls app('livewire')->getInstance($name, $id)

    public static function fromInitialRequest($name, $id)
    {
        return tap(new static, function ($instance) use ($name, $id) {
            $instance->instance = app('livewire')->getInstance($name, $id);   <====

where $name is 'counter' and $id is gibberish

I'm assuming that is getting into Laravel itself, where I don't want to go.

Am I'm going to have to give up on this utility? That will make me very sad.

**Comment by joshhanley** [comment]
@chuckgriffith2 Livewire has been compatible with Laravel 9 since version v.2.10.0.

Hope this helps!
